gateway/src/chains/avalanche/avanlanche_tokens.json

```diff
@@ -0,0 +1,1565 @@
+      },
+      {
+        "chainId": 43114,
+        "address": "0x783C08b5F26E3daf8C4681F3bf49844e425b6393",
+        "decimals": 18,
+        "name": "Avaware USD",
+        "symbol": "AUSD",
+        "logoURI": "https://raw.githubusercontent.com/pangolindex/tokens/main/assets/43114/0x783C08b5F26E3daf8C4681F3bf49844e425b6393/logo_24.png"
+      },
+      {
+        "chainId": 43114,
+        "address": "0x346A59146b9b4a77100D369a3d18E8007A9F46a6",
+        "decimals": 18,
+        "name": "AVAI",
+        "symbol": "AVAI",
+        "logoURI": "https://raw.githubusercontent.com/pangolindex/tokens/main/assets/43114/0x346A59146b9b4a77100D369a3d18E8007A9F46a6/logo_24.png"
+      },
+      {
+        "chainId": 43114,
+        "address": "0xD24C2Ad096400B6FBcd2ad8B24E7acBc21A1da64",
+        "decimals": 18,
+        "name": "Frax",
+        "symbol": "FRAX",
+        "logoURI": "https://raw.githubusercontent.com/pangolindex/tokens/main/assets/43114/0xD24C2Ad096400B6FBcd2ad8B24E7acBc21A1da64/logo_24.png"
+      },
+      {
+        "chainId": 43114,
+        "address": "0x5c49b268c9841AFF1Cc3B0a418ff5c3442eE3F3b",
+        "decimals": 18,
+        "name": "miMatic",
+        "symbol": "MAI",
+        "logoURI": "https://raw.githubusercontent.com/pangolindex/tokens/main/assets/43114/0x5c49b268c9841AFF1Cc3B0a418ff5c3442eE3F3b/logo_24.png"
+      },
+      {
+        "chainId": 43114,
+        "address": "0x026187BdbC6b751003517bcb30Ac7817D5B766f8",
+        "decimals": 18,
+        "name": "Defrost Finance H2O",
+        "symbol": "H2O",
+        "logoURI": "https://raw.githubusercontent.com/pangolindex/tokens/main/assets/43114/0x026187BdbC6b751003517bcb30Ac7817D5B766f8/logo_24.png"
+      },
+      {
+        "chainId": 43114,
+        "address": "0xB97EF9Ef8734C71904D8002F8b6Bc66Dd9c48a6E",
+        "decimals": 6,
+        "name": "USD Coin",
+        "symbol": "USDC",
+        "logoURI": "https://raw.githubusercontent.com/pangolindex/tokens/main/assets/43114/0xB97EF9Ef8734C71904D8002F8b6Bc66Dd9c48a6E/logo_24.png"
+      },
+      {
+        "chainId": 43114,
+        "address": "0x6feFd97F328342a8A840546A55FDcfEe7542F9A8",
+        "decimals": 18,
+        "name": "agEUR",
+        "symbol": "agEUR",
+        "logoURI": "https://raw.githubusercontent.com/pangolindex/tokens/main/assets/43114/0x6feFd97F328342a8A840546A55FDcfEe7542F9A8/logo_24.png"
+      },
+      {
+        "chainId": 43114,
+        "address": "0x260Bbf5698121EB85e7a74f2E45E16Ce762EbE11",
+        "decimals": 6,
+        "name": "Axelar Wrapped UST",
+        "symbol": "axlUST",
+        "logoURI": "https://raw.githubusercontent.com/pangolindex/tokens/main/assets/43114/0x260Bbf5698121EB85e7a74f2E45E16Ce762EbE11/logo_24.png"
+      },
+      {
+        "chainId": 43114,
+        "address": "0x9702230A8Ea53601f5cD2dc00fDBc13d4dF4A8c7",
+        "decimals": 6,
+        "name": "TetherToken",
+        "symbol": "USDt",
```

Comment:
Id like to suggest if we can change the symbol from `USDt` to `USDT` to fix the issue on warning prompts from the rate oracle conversion? I'm under the impression as long as we are using the correct token address it should work. Please see screenshot below, Thanks

![image](https://user-images.githubusercontent.com/73840223/208016747-a2da32b6-883a-451f-9882-53c17c312b39.png)

```suggestion
        "symbol": "USDT",
```